### PR TITLE
fix(levm): change CI check for EFTests because London doesn't pass 100% now

### PR DESCRIPTION
**Pull request by JereSalo**
**Motivation**

- Fix CI 

**Description**

- London tests don't pass 100% because new tests have been added and it seems that there is an edge case we are not currently passing. For now I wanted to disable the check that sees if all tests from Prague to London passed and set it to look only for Prague to Paris.
- Added `workflow_dispatch` to this workflow so that we can run it manually.

Closes #issue_number

### CODE REVIEW

**Review by ilitteri — APPROVED** [review]

**Review by edg-l — APPROVED** [review]

File: .github/workflows/pr-main_levm.yaml [hunk 1]
```diff
@@ -118,7 +119,7 @@ jobs:
       - name: Check EF-TESTS from London to Prague is 100%
```
Comment (fmoletta) [inline .github/workflows/pr-main_levm.yaml:119]:
```suggestion
      - name: Check EF-TESTS from Paris to Prague is 100%
```

**Review by fmoletta — APPROVED** [review]

File: .github/workflows/pr-main_levm.yaml (hunk 1 above)
Comment (JereSalo) [reply .github/workflows/pr-main_levm.yaml]: Good catch! https://github.com/lambdaclass/ethrex/pull/2568/commits/e4acd32fdd9de0388769d3343e25752c46a32701

**Comment by ilitteri** [comment]
It seems that we have a Hive diff with `main`.
